tct/Cargo.toml

```diff
@@ -6,18 +6,35 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+penumbra-proto = { path = "../proto/" }
+
 derivative = "2"
 once_cell = "1"
 blake2b_simd = "1"
 hex = "0.4"
 hash_hasher = "2"
 thiserror = "1"
 serde = { version = "1.0", features = ["derive"] }
+parking_lot = "0.12"
 ark-ff = { git = "https://github.com/penumbra-zone/algebra", branch = "ours" }
 ark-serialize = { git = "https://github.com/penumbra-zone/algebra", branch = "ours" }
 poseidon377 = { git = "https://github.com/penumbra-zone/poseidon377" }
 decaf377 = { git = "https://github.com/penumbra-zone/decaf377" }
+# this allows us to implement Decode for various types without orphan rule issues
+sqlx = { version = "0.5.9", features = ["postgres"], optional = true }
```

Comment:
We should not include `sqlx` here, since we've just finished standardizing on using our canonical, proto-based serialization for persistent storage.

Reply:
Sqlx has been removed in the tct-integration-tct-only branch.